test/powershell/Modules/Microsoft.PowerShell.Security/CertificateProvider.Tests.ps1

```diff
@@ -44,11 +44,11 @@ Describe "Certificate Provider tests" -Tags "CI" {
     }
 
     Context "Get-Item tests" {
-        it "Should be able to get a certificate store, path: <path>" -TestCases $testLocations {
+        it "Should be able to get a certificate store, path: <path>" -TestCases $testLocations -Pending:$true {
```

Comment:
I think we should exclude this from the PR and make a fix as proposed in related issue.

Reply:
We cannot exclude this without fixing the issue. I would prefer to fix the underlying issue in a seperate PR.

Reply:
We should have working test after the _style_ change. If this is not so, we can not merge. Please revert the change and fix the behavior and the style in follow PR.

Reply:
The test was broken before this PR so why should this block merge? I have WIP PR #11590 to fix the underlying issue but I am finding it hard to debug :(

Reply:
Our policy is (1) to split functional and style/formatting changes, (2) keep tests and code in right (working) state after any commit.
Before you change the test is passed so we should skip the "style" change and discuss in #11590.

Reply:
@iSazonov The test is broken without this PR so it should be marked as pending in any case.

Maybe this helps to explain:
```powershell
$result = $null
$result | Should -Not -Be null  # passes
$result | Should -Not -Be $null # fails
```

Reply:
@xtqqczze After you revert the change we will merge and you can continue the work in #11590